Patches/MorrowRim - Ashlands Swamp/SwampInsec.xml

```diff
@@ -0,0 +1,273 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	<Operation Class="PatchOperationFindMod">
+		<mods>
+		<li>MorrowRim - Ashlands Swamp</li>
+		</mods>
+
+		<match Class="PatchOperationSequence">
+			<operations>
+
+			<li Class="PatchOperationAddModExtension">
+				<xpath>/Defs/ThingDef[defName="MorrowRim_SwampStrider"]</xpath>
+				<value>
+					<li Class="CombatExtended.RacePropertiesExtensionCE">
+						<bodyShape>Humanoid</bodyShape>
```

Comment:
They are extremely tall with long narrow legs? Definitely want to empathise their height here.

Reply:
Hmm. This may actually be a case where a new body type is in order, since it doesn't really fit any of the existing very well. I'll see if I can put one together and add it to one of my open PRs.

Reply:
Well, humanoids are tall and narrow, i say it fits them pretty well. It is only a xml bodyshape, it is not like anyone would see them  having "humanoid" bodyshape ingame.

Reply:
I've added a `QuadrupedTall` bodyShape in #723 which should suit this general shape of creature nicely.

Reply:
> I've added a `QuadrupedTall` bodyShape in #710 which should suit this general shape of creature nicely.

Quadruped tall has the exact dimension as vehicles(as width as it is tall), these strider are not square, they are very narrow and very tall. They are not in any way shape or form, as wide as they are tall.